content/recipes/nestia.md

```diff
@@ -0,0 +1,243 @@
+
+```typescript
+export interface IBbsArticle {
+    /**
+     * @format uuid
+     */
+    id: string;
+
+    writer: IBbsArticle.IWriter;
+
+    /**
+     * @minItems 1
+     */
+    contents: IBbsArticle.IContent[];
+}
+export namespace IBbsArticle {
+    export interface IWriter {
+        /**
+         * @minLength 3
+         */
+        name: string;
+
+        /**
+         * @format email
+         */
+        email: string;
+
+        /**
+         * @pattern ^0[0-9]{7,16}
+         */
+        mobile: string;
+
+        /**
+         * @minimum 18
+         */
+        age: number;
+    }
+}
+```
+
+
+
+#### `@nestia/sdk`
+
+Automatic *SDK* and *Swagger* generator for `@nestia/core`.
+
+##### Usage
+
+```bash
+# BASIC COMMAND
+npx nestia <sdk|swagger> <source_directories_or_patterns> \
+    --exclude <exclude_directory_or_pattern> \
+    --out <output_directory_or_file>
+
+# EXAMPLES
+npx nestia sdk "src/**/*.controller.ts" --out "src/api"
+npx nestia swagger "src/controllers" --out "dist/swagger.json"
+```
+
+You can generate SDK (Software Development Kit) library or Swagger Documents from above commands.
+
+If you've configured `nestia.config.ts` file, you can generate them much easily like below. About the configuration file, read [Guide Documents - Configuration](https://github.com/samchon/nestia/wiki/Configuration)
+
+```bash
+npx nestia sdk
+npx nestia swagger
+```
+
+##### Demonstration
+
+When you run `npx nestia sdk` command, `@nestia/sdk` will generate below code.
+
+```typescript
+import { Fetcher, IConnection } from "@nestia/fetcher";
+import { IBbsArticle } from "../../../structures/IBbsArticle";
+
+/**
+ * Store a new content.
+ * 
+ * @param input Content to store
+ * @returns Newly archived article
+ */
+export function store(
+    connection: api.IConnection, 
+    input: IBbsArticle.IStore
+): Promise<IBbsArticle> {
+    return Fetcher.fetch(
+        connection,
+        store.ENCRYPTED,
+        store.METHOD,
+        store.path(),
+        input
+    );
+}
+export namespace store {
+    export const METHOD = "POST" as const;
+    export function path(): string {
+        return "/bbs/articles";
+    }
+}
+```
+
+With the SDK library, client developers would get take advantages of TypeScript like below. 
+
+If you want to learn how to distribute SDK library, visit and read [Guide Documents - Distribution](https://github.com/samchon/nestia/wiki/Distribution).
+
+```typescript
+import api from "@bbs-api";
+import typia from "typia";
+
+export async function test_bbs_article_store(connection: api.IConnection) {
+    const article: IBbsArticle = await api.functional.bbs.articles.store(
+        connection,
+        {
+            name: "John Doe",
+            title: "some title",
+            content: "some content",
+        }
+    );
+    typia.assert(article);
+    console.log(article);
+}
+```
```

Comment:
I'm iffy on if we want to show "client" code. I get the draw for it, just not sure as I don't think we explicitly show client code anywhere else in the docs.

Reply:
I can't come up with a good way to express the SDK in a short and concise way. 

Do you have any good idea?

Reply:
I think just linking over to your own docs for how to use the SDK would be fine. Again, I'm not sure n showing client code or not, but I do get why you'd want to show this off. It's a hard choice :thinking:

Reply:
Okay, then I'll erase the last example code like below review comment